pep-0696.rst

```diff
@@ -507,6 +529,19 @@ convenient, could have a ``TypeVarLike`` with no default follow a
 This would have also been a breaking change for a small number of cases
 where the code relied on ``Any`` being the implicit default.
 
+Allowing ``TypeVarLike``\ s with defaults to be used in function signatures
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+A previous version of this PEP allowed ``TypeVarLike``\ s to be used in
+function signatures. This was removed for the rationale described In
+:ref:`function-defaults`. Hopefully, this can be added in the future if
+a way to get the runtime value of a type parameter is added.
+
+Allowing ``TypeVarLikes`` from outer scopes in ``default``
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+This was deemed too niche a feature to be worth the added complexity, if
+any cases arise where this is needed, it can be added in a future PEP.
```

Comment:
```suggestion
This was deemed too niche a feature to be worth the added complexity.
If any cases arise where this is needed, it can be added in a future PEP.
```

Fix grammar error (major comma splice)